examples/Backtesting/botorch_analytical.py

```diff
@@ -119,13 +107,24 @@
 results = simulate_scenarios(
     scenarios,
     WRAPPED_FUNCTION,
-    batch_size=3,
+    batch_size=BATCH_SIZE,
     n_doe_iterations=N_DOE_ITERATIONS,
     n_mc_iterations=N_MC_ITERATIONS,
 )
 
-# The following lines plot the results and save the plot in run_analytical.png
-
-sns.lineplot(data=results, x="Num_Experiments", y="Target_CumBest", hue="Scenario")
-plt.gcf().set_size_inches(24, 8)
-plt.savefig("./run_analytical.png")
+# We use the plotting utility to create plots.
+
+# ```{note}
```

Comment:
this is only useful for us and not useful for the reader so please remove

Reply:
You mean the note in general? My intention here was to provide information about why we do not use `__file__` since this is a question users might have. If we however do not care about that, I am more than happy to either completely remove the note or just have an in-line comment in the style of `# not using __file__ here for technical reasons`

Reply:
I dont think any user will wonder more than they would eg wonder `why did they use pyplot and not ggplot?` Is simply an alternative, no need to justify

Reply:
Removed the note.